packages/react-textarea/src/components/TextArea/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './TextArea';
+export * from './TextArea.types';
+export * from './useTextArea';
+export * from './useTextAreaStyles';
+export * from './renderTextArea';
```

Comment:
nit: Order these alphabetically

```suggestion
export * from './renderTextArea';
export * from './useTextArea';
export * from './useTextAreaStyles';
```